README.md

```diff
@@ -25,7 +25,7 @@ structures:
 * [Bifunctor](#bifunctor)
 * [Profunctor](#profunctor)
```

Comment:
It does support multiple output formats, including `plain`. `plain` gives you the size of the graph, a sequence of nodes with positions, and a sequence of edges with position if you want to parse the coordinates into an ascii thing. It looks like a regular language, so probably not too hard to parse.